x-pack/platform/plugins/shared/alerting/server/application/rule/methods/run_soon/schemas/run_soon_params_schema.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const runSoonParamsSchema = schema.object({
+  id: schema.string(),
+  force: schema.maybe(schema.boolean({ defaultValue: false })),
```

Comment:
I don't know how it actually works, but does it make sense to have `maybe` **and** `defaultValue`? Isn't this redundant?